### DISCUSSION TITLE
Character limit changes to text yellow at 260

### DISCUSSION THREAD

**Discussion by KyleJSh** [deso-protocol/dips#136] [Medium CIPs]
Even though the character limit was increased to 560, the yellow warning text color still appears at 260, way too early.

==

![Screen Shot 2021-08-22 at 9 20 48 AM](https://user-images.githubusercontent.com/30233330/130362558-6c933a18-31b5-4e31-97ca-ab79bd249608.png)

**Comment by tijno** [comment]
Its a bug - it was not expanded when the character limit was raised. Ill submit a PR now to raise it.

**Reply by tijno** [reply]
https://github.com/bitclout/frontend/pull/408

done

@KyleJSh
